### PR TITLE
 Change paths of two buttons on conference show page and fixes a typo

### LINKED ISSUE #1813: splashpage's submit request for track should redirect to track#new
On Splashpage submit request for track is redirecting user to program/tracks which is a empty page initially with new track request button, therefore the button on splashpage should redirect user to track#new

### PR DESCRIPTION
**Pull request by AnkushMalik**
**Checklist**

- [x] I have read the [Contribution & Best practices Guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md).
- [x] My branch is up-to-date with the upstream `master` branch.
- [x] The tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works(if appropriate).
- [x] I have added necessary documentation (if appropriate).

**Short description of what this resolves/which [issues](https://github.com/openSUSE/osem/issues) does this fix?:**

- Closes #1813

**Changes proposed in this pull request:**

- Update path of track submission btn to track#new and proposal submission to proposal#new

### CODE REVIEW

File: app/views/conferences/show.html.haml [hunk 1]
```diff
@@ -29,7 +29,7 @@
     - if @conference.registration_open?
       = render 'registration', conference: @conference,
         registration_period: @conference.registration_period,
-        tickets: @tickets, conference_id: @conference_id.short_title
+        tickets: @tickets, conference_id: @conference.short_title
```
Comment (bear454) [inline app/views/conferences/show.html.haml]: Nice catch! 👍

File: db/schema.rb [hunk 2]
```diff
@@ -215,8 +215,8 @@
     t.integer  "schedule_id"
     t.integer  "room_id"
     t.datetime "start_time"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.datetime "created_at",                 null: false
```
Comment (bear454) [inline db/schema.rb]: This is unrelated... can you remove it from the PR ?

File: app/views/conferences/_call_for_tracks.haml [hunk 3]
```diff
@@ -15,5 +15,5 @@
         left!
     %p.cta-button
       = link_to("Submit your request for track",
-        conference_program_tracks_path(conference_id),
+        new_conference_program_track_path(conference_id),
```
Comment (bear454) [inline app/views/conferences/_call_for_tracks.haml]: JFYI; this conflicts with/supercedes #1865.

Comment (AnkushMalik) [reply app/views/conferences/_call_for_tracks.haml]: that was unresponsive for very long so i thought may be could resolve that issue,
i will comment from the next time. :)

**Comment by bear454** [comment]
@AnkushMalik can you please update the PR for just the issue being resolved (the subject is no longer accurate because of #1925), and rebase the code to only fix #1813 ?

**Comment by differentreality** [comment]
@bear454 FYI I am not sure we should go ahead with the change 
https://github.com/openSUSE/osem/issues/1813#issuecomment-355088888

**Comment by bear454** [comment]
@differentreality let's move this discussion to the issue...

**Comment by bear454** [comment]
@differentreality do you approve, then?

**Comment by AnkushMalik** [comment]
i will update/rebase the PR now. :)
